### PR TITLE
LPS-78295 Add repeatable icon in all repeatable fields

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder/form_builder.js [hunk 1]
```diff
@@ -805,6 +817,7 @@ AUI.add(
 						var instance = this;
 
 						instance._eventHandlers.push(
+							A.one('.' + CSS_LAYOUT_BUILDER_CONTAINER).delegate('hover', A.bind('onHoverColumn', instance), '.col', instance),
```
Comment (pedroqueiroz) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder/form_builder.js]: hey @alinedoleron , why is this repeated?

Comment (alinedoleron) [reply modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder/form_builder.js]: Weird, I did't even use this line on my fix. I'd duplicated by mistake. I'll remove it.

**Comment by rafaprax** [comment]
Pull request submitted to https://github.com/rafaprax/liferay-portal/pull/552. [See changes here.](https://github.com/rafaprax/liferay-portal/compare/fd92fc2e19e4~1...fd92fc2e19e4) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by rafaprax** [comment]
ci:stop

**Pull request closed without merging**